### PR TITLE
Newsletter update Nov 2018

### PR DESCRIPTION
**Pull request by cathywicks**

### CODE REVIEW

File: 2018-11/index.md [hunk 1]
```diff
@@ -4,69 +4,69 @@ title: BeagleBoard.org Enables Success
...
+   image: https://hackster.imgix.net/uploads/attachments/612564/img_e2371_smB8gEhktG.JPG
+ - title: BeagleBone® Green Wireless Irrigation Control
+   link: https://beagleboard.org/p/Greg-R/beaglebone-green-wireless-irrigation-control-ce7c4b
+   body: A practical project creating a wireless servo/controller for a home irrigation system.
+   image: https://hackster.imgix.net/uploads/attachments/313268/cover_photo_Iijm9fz1MC.jpg?auto=compress%2Cformat&w=900&h=675&fit=min
+ - title: BoneScript
+   link: http://beagleboard.org/Support/BoneScript
+   body: Easy to get started with Physical computing library in JavaScript for Node.JS and the browser
+   image: https://hackster.imgix.net/uploads/attachments/362165/file.png?auto=compress%2Cformat&w=900&h=675&fit=min
+ - title: PocketBeagle® Magic 8-ball game with Mikro Click Boards™
```
Comment (jadonk) [inline 2018-11/index.md:68]: I suspect this trademark/branding usage is not quite right.
```suggestion
 - title: PocketBeagle® Magic 8-ball game with MikroElektronika’s click boards™
```

File: 2018-11/index.md [hunk 2]
```diff
@@ -4,69 +4,69 @@ title: BeagleBoard.org Enables Success
...
+ - title: SCUTTLE Mobile Utility Robot 
+   link: https://beagleboard.org/p/112547/scuttle-mobile-utility-robot-from-texas-a-m-d6c2b7
+   body: Educational Platform from Texas A&M Mechatronics Engineering Technology Program using BeagleBone® Blue
+   image: https://hackster.imgix.net/uploads/attachments/612564/img_e2371_smB8gEhktG.JPG
+ - title: BeagleBone® Green Wireless Irrigation Control
+   link: https://beagleboard.org/p/Greg-R/beaglebone-green-wireless-irrigation-control-ce7c4b
+   body: A practical project creating a wireless servo/controller for a home irrigation system.
+   image: https://hackster.imgix.net/uploads/attachments/313268/cover_photo_Iijm9fz1MC.jpg?auto=compress%2Cformat&w=900&h=675&fit=min
+ - title: BoneScript
+   link: http://beagleboard.org/Support/BoneScript
```
Comment (jadonk) [inline 2018-11/index.md:65]:
```suggestion
   link: https://bbb.io/+b90401
```

If that link doesn't look nice enough, I've given you editing rights to the project.

File: 2018-11/index.md [hunk 3]
```diff
@@ -4,69 +4,69 @@ title: BeagleBoard.org Enables Success
...
- - title: Smart Mirror
-   link: http://bbb.io/+559755
-   body: Students at University of Texas built their own custom mirror with BeagleBone® Black and parts from the hardware store.
-   image: https://beagleboard.github.io/newsletter/static/images/newsletter-2018-09_0007.JPG
+ - title: SCUTTLE Mobile Utility Robot 
+   link: https://beagleboard.org/p/112547/scuttle-mobile-utility-robot-from-texas-a-m-d6c2b7
+   body: Educational Platform from Texas A&M Mechatronics Engineering Technology Program using BeagleBone® Blue
+   image: https://hackster.imgix.net/uploads/attachments/612564/img_e2371_smB8gEhktG.JPG
+ - title: BeagleBone® Green Wireless Irrigation Control
+   link: https://beagleboard.org/p/Greg-R/beaglebone-green-wireless-irrigation-control-ce7c4b
```
Comment (jadonk) [inline 2018-11/index.md:61]:
```suggestion
   link: https://bbb.io/+ce7c4b
```

You don't HAVE to use these short links, but I think they are kinda handy and nice to promote. The format is https://bbb.io/+ followed by the last 6 characters in the project link.

File: 2018-11/index.md [hunk 4]
```diff
@@ -4,69 +4,69 @@ title: BeagleBoard.org Enables Success
...
-   body: Check out the options discussed in this thread to help with the choices.
- - title: Back to School with Embedded Linux on BeagleBone®
-   link: https://groups.google.com/forum/embed/?place=forum/beagleboard&showsearch=true&showpopout=true&showtabs=false&hideforumtitle=true&parenturl=https%3A%2F%2Fbeagleboard.org%2Fdiscuss%2F%3Fplace%3Dtopic%252Fbeagleboard%252FoMCYbQgzjQo%252Fdiscussion#!category-topic/beagleboard/beaglebone-black/XJeKUBAzivI
-   body: Check out with Prof Yoder’s course with materials posted in this thread.
- - title: Qt5 "hello world" how-to on BeagleBone Debian
-   link: https://groups.google.com/forum/embed/?place=forum/beaglebone&showsearch=true&showpopout=true&showtabs=false&hideforumtitle=true&parenturl=https%3A%2F%2Fbeagleboard.org%2Fdiscuss%2F%3Fplace%3Dtopic%2Fbeagleboard%2FoMCYbQgzjQo%2Fdiscussion#!category-topic/beagleboard/software/bGuAcHm0atc
-   body: Need to use Qt on an embedded system – check this post for latest getting started
+ - title: Fresh Debian 9.5 Images
+   link: https://beagleboard.org/blog/2018-10-10-fresh-debian-9-5-images
```
Comment (jadonk) [inline 2018-11/index.md:47]:
```suggestion
   link: https://bbb.io/@2019
```

You don't have to use the shortlinks, but they are handy and I'd like to promote them. You can grab the shortlinks off of the wp-admin posts page by clicking "Get Shortlink".

File: 2018-11/index.md [hunk 5]
```diff
@@ -4,69 +4,69 @@ title: BeagleBoard.org Enables Success
...
- - title: BeagleBoard.org 10th Anniversary &num;BringAHack Event, New York City
-   date: September 21, 2018 
-   link: https://www.eventbrite.com/e/bringahack-new-york-with-beagleboardorg-before-maker-faire-tickets-49521195228
- - title: World MakerFaire, New York City
-   date: September 22-23, 2018
-   link: https://beagleboard.org/blog/2018-09-05-join-beagleboard-org-at-world-maker-faire-in-new-york-sept-21-23
+ - title: Embedded World 2019 Exhibition & Conference, Nuremberg, Germany
+   date: February 26-28, 2019 
+   link: https://www.embedded-world.de/en/ausstellerprodukte/embwld19/exhibitor-38827067/beagleboard-org-foundation
+ - title: Linux Jumpstart with PocketBeagle Hands-On Workshop at Embedded World
```
Comment (jadonk) [inline 2018-11/index.md:12]: Did you mean to use the phrase "Jumpstart"? I don't think I saw that before. That has typically been used by Bill Gatliff for his courses.

File: 2018-11/index.md [hunk 6]
```diff
@@ -4,69 +4,69 @@ title: BeagleBoard.org Enables Success
...
+   date: February 27, 2019 
+   link: https://www.embedded-world.de/en/ausstellerprodukte/embwld19/exhibitor-38827067/beagleboard-org-foundation
+ - title: Want to schedule a meeting with Jason or Christine?
+   date: Schedule online
+   link: https://beagleboard.org/about
```
Comment (jadonk) [inline 2018-11/index.md:17]:
```suggestion
   link: https://www.embedded-world.de/en/ausstellerprodukte/embwld19/exhibitor-38827067/beagleboard-org-foundation#exhibitorcontact
```

At least use the direct calendly link to try to save folk's time. I need to explore if I should update calendly.com, but I think maybe we should just use Embedded World's system. I just sent a test message.

File: 2018-11/index.md (hunk 5 above)
Comment (jadonk) [reply 2018-11/index.md:12]: Official name is "Getting Started with PocketBeagle from BeagleBoard.org - A Hands-On Coding with Embedded Linux Workshop"

File: 2018-11/index.md [hunk 7]
```diff
@@ -4,69 +4,69 @@ title: BeagleBoard.org Enables Success
...
-   link: https://www.eventbrite.com/e/bringahack-new-york-with-beagleboardorg-before-maker-faire-tickets-49521195228
- - title: World MakerFaire, New York City
-   date: September 22-23, 2018
-   link: https://beagleboard.org/blog/2018-09-05-join-beagleboard-org-at-world-maker-faire-in-new-york-sept-21-23
+ - title: Embedded World 2019 Exhibition & Conference, Nuremberg, Germany
+   date: February 26-28, 2019 
+   link: https://www.embedded-world.de/en/ausstellerprodukte/embwld19/exhibitor-38827067/beagleboard-org-foundation
+ - title: Linux Jumpstart with PocketBeagle Hands-On Workshop at Embedded World
+   date: February 27, 2019 
+   link: https://www.embedded-world.de/en/ausstellerprodukte/embwld19/exhibitor-38827067/beagleboard-org-foundation
```
Comment (jadonk) [inline 2018-11/index.md:14]:
```suggestion
   link: https://www.embedded-world.eu/program.html#48912
```

**Review by jadonk — APPROVED** [review]

File: 2018-11/index.md (hunk 5 above)
Comment (cathywicks) [reply 2018-11/index.md:12]: On the name - the Embedded World coordinator asked if they could shorten the name of the workshop and made that recommendation.   " Linux Jumpstart with PocketBeagle from BeagleBoard.org"  see this page and scroll down.  https://www.embedded-world.eu/program.html

File: 2018-11/index.md (hunk 4 above)
Comment (cathywicks) [reply 2018-11/index.md:47]: okay will do next time

File: 2018-11/index.md (hunk 3 above)
Comment (cathywicks) [reply 2018-11/index.md:61]: gotcha

File: 2018-11/index.md (hunk 2 above)
Comment (cathywicks) [reply 2018-11/index.md:65]: that's a good one, thank you for finding it

File: 2018-11/index.md (hunk 6 above)
Comment (cathywicks) [reply 2018-11/index.md:17]: sounds good

**Pull request merged**